pkg/graveler/model.go

```diff
@@ -17,7 +17,6 @@ const (
 	settingsPrefix         = "settings"
 	importsPrefix          = "imports"
 	repoMetadataPrefix     = "repo-metadata"
-	pullRequestsPrefix     = "pulls"
```

Comment:
Why treating this different than the other APIs?
I think it's better to be consistent, even if it's only used outside of this file.

Reply:
The main reason is because of secondary indices but also because this is not really part of the graveler model.
We added it under ref manager to increase velocity but optimally it should be a separate service once it's out of MVP